### ISSUE TITLE
Version 5

### ISSUE THREAD

**Issue by DRYN07** [nyxiereal/XToolbox#44]
Could you perhaps share some updates regarding the new version? Or is there any platform where we can track the news about v5?

Thanks :)

**Comment by nyxiereal** [comment]
I stopped using Windows after releasing the last version, I'm now on Arch Linux and couldn't be happier. I'll release the (probably) last update to XTB (hopefully) before the end of this year. Sorry for the late reply, I've been really sick lately.

I'll close this issue when v5 releases :3

**Comment by gigi2006** [comment]
i really hope that you will develop a new version... maybe there will be a fork or something after that? your tool was just great!


> I stopped using Windows after releasing the last version, I'm now on Arch Linux and couldn't be happier. I'll release the (probably) last update to XTB (hopefully) before the end of this year. Sorry for the late reply, I've been really sick lately.
> 
> I'll close this issue when v5 releases :3

**Comment by nyxiereal** [comment]
update about the state of the project: i will try to make v5 in another form, like a database of tweaks taken from other software, all in one place (with attribution ofc). But before i start working on that i will do some bugfixes and release v4.3.

Tldr: v5 will be a database of tweaks, v4.3 will release before that

**Comment by DRYN07** [comment]
That is an amazing idea, thank you for your efforts.
